rust/processor/src/worker.rs

```diff
@@ -351,7 +359,8 @@ impl Worker {
             let mut transactions_batches = vec![];
             let mut last_fetched_version = batch_start_version as i64 - 1;
             for task_index in 0..concurrent_tasks {
-                let receive_status = match task_index {
+                let receive_status: Result<(u64, Vec<Transaction>), TryRecvError> = match task_index
```

Comment:
I don't think that this is intentional? Can we remove?

Reply:
oops